content/reference/api/engine/_index.md

```diff
@@ -3,28 +3,7 @@ title: Develop with Docker Engine API
 description: Learn how you can use Docker Engine API and SDKs in the language of your choice.
 keywords: developing, api, Docker Engine API, API versions, SDK versions
 aliases:
-- /engine/reference/api/
-- /engine/reference/api/docker_remote_api/
```

Comment:
Could we have wildcard redirects for these (if the number of redirects is the issue)?